package.json

```diff
@@ -37,10 +39,10 @@
     "eslint-plugin-jest": "^22.5.1",
     "eslint-plugin-jsx-a11y": "^6.1.1",
     "eslint-plugin-react": "^7.11.1",
-    "faker": "^4.1.0",
     "html-webpack-plugin": "^3.2.0",
     "jest": "^26.0.1",
     "mini-css-extract-plugin": "^0.4.0",
+    "prettier": "^2.0.5",
```

Comment:
This isn't working out. Prettier doesn't respect the ESLint configuration. Prettier cannot be configured to respect the ESLint configuration. smh